lib/nest-thermostat-accessory.js

```diff
@@ -238,7 +238,7 @@ NestThermostatAccessory.prototype.getTargetTemperature = function () {
 NestThermostatAccessory.prototype.getCoolingThresholdTemperature = function () {
   switch (this.device.hvac_mode) {
   case "eco":
-    return this.device.eco_temperature_high_c;
+    return this.device.eco_temperature_high_c ? this.device.eco_temperature_high_c : this.device.away_temperature_high_c;
```

Comment:
Same thing, slightly simpler:
`return this.device.eco_temperature_high_c || this.device.away_temperature_high_c;`

Can you add an in-file comment explaining the need for this fallback?